### PR TITLE
Redesign inactive user

### PR DESCRIPTION
**Pull request by MaianaB**
**Feature/Bug description:** When user is redirect to user inactive page using mobile he can't go back to login page.

**Solution:** Added go back button on small displays.

**TODO/FIXME:** n/a

captura de tela de 2018-10-04 10-17-35
captura de tela de 2018-10-04 10-17-50
captura de tela de 2018-10-04 10-19-56
captura de tela de 2018-10-04 10-20-06
captura de tela de 2018-10-04 10-21-39

### CODE REVIEW

File: frontend/auth/signup.html [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <div layout="row" layout-align="center center">
   <md-button class="md-fab md-primary" aria-label="Retornar" 
-    style="margin-right: -27px; width: 65px; height: 65px;" md-colors="{background: 'light-green-600'}"
+    style="margin-right: -27px;" md-colors="{background: 'light-green-600'}"
```
Comment (ruanres) [inline frontend/auth/signup.html]: move style from template to css file

Comment (ruanres) [reply frontend/auth/signup.html]: The same in the other cases

File: frontend/user/user_inactive.html [hunk 2]
```diff
@@ -1,15 +1,14 @@
 <md-content layout-fill layout="column" md-colors="{background: 'teal-500'}" layout-align="center center">
-    <div layout="row" layout-align="center center">
-        <md-button class="md-fab md-primary" aria-label="Retornar" ng-click="userInactiveCtrl.logout()"
-            style="margin-right: -35px;" md-colors="{background: 'light-green-500'}" hide-xs>
+    <div layout="row" layout-align="center center" style="margin: 0 1em 0 1em">
```
Comment (Luiz-FS) [inline frontend/user/user_inactive.html]: Move to css file

File: frontend/auth/signup.html (hunk 1 above)
Comment (MaianaB) [reply frontend/auth/signup.html]: This page wasn't made the refactoring, I fixed only the button.

**Pull request merged**